BUILD.md

```diff
@@ -0,0 +1,16 @@
+# Build
+
+## Dependencies
+
+- [jEnv](http://www.jenv.be/)
```

Comment:
If you want to support 1.7 and 1.8 we need a way to test it. Any developer who has some other means to provide both versions locally, great. No need to use the script.